### PR TITLE
[quick merge] Revert "Fixes overexcited TTV reporting"

### PR DESCRIPTION
**Pull request by MrsTonedOne**
Revert's #9745

This prevented bomb openings from getting logged or admins from getting notified when somebody opened a bomb valve. I had to look at href logs to find a griffer

(Protip: the volume of the tanks will always be 70, no matter what is in it.)

### CODE REVIEW

**Comment by Incoming5643** [comment]
This was merged months ago, it just became a problem now?

Did someone intentionally manage to make a bomb that had equivalent pressure before mixing and yet could still explode?

**Comment by MrStonedOne** [comment]
its been an issue for months, but admins never mentioned anything to me until a little over a week ago, and i hadn't gotten around to trying to figure it out until yesterday.

I asked remie about it, and this is what she said:

Remie: hey MrSleepyOne
Remie: I think I figured it out haha
Remie: it's deffinitely incoming's spam fix
Remie: he checks the volume of both tanks... which doesn't change
Remie: gases inside the tank change their volumes and stuff, but he checks the TANKS volumes
Remie: which for plasma/oxy tanks will always be 70 haha.

You check the volume of the tanks.

This is ALWAYS 70.

**Comment by as334** [comment]
Actually if you use emergency tanks it can be 3, and if you use those yellow ones it can be 7.

**Comment by Incoming5643** [comment]
I was gonna say I just checked the logs and I was still seeing the occasional opened valve message, so it's not every case.

No one does this but you can also use jetpacks in TTV bombs, they might have a different volume too.

</img>

**Comment by RemieRichards** [comment]
@Incoming5643 
one, it was merged A month ago, no months ago, second, the issue is that you checked the volume of the tank... this is not changed during explosions/pressure change/equations AT ALL, it's how much the tank can hold, not how much it's actually holding, had you checked a variable on the gas mixture datum INSIDE ```tank1``` and ```tank2``` then this would probably have worked.

@as334 hence plasma/oxy tanks, I should have specified but yes, I meant the tanks there not the gases. 

Essentially this code prevented the logging whenever the two tanks were of the same size, which for the 3 most common TTV setups oxy+oxy/oxy+plasma/plasma+plasma, that's 70+70, which is always the same.

tl;dr the check should be looking at variables on the gas mixtures in the tanks, not on the tanks.
